cantools/database/can/formats/dbc.py

```diff
@@ -1006,7 +1022,7 @@ def _load_value_tables(tokens):
 
     for value_table in tokens.get('VAL_TABLE_', []):
         name = value_table[1]
-        choices = {int(number): NamedSignalValue(number, text) for number, text in value_table[2]}
+        choices = {int(number): NamedSignalValue(int(number), text) for number, text in value_table[2]}
```

Comment:
hm, I'm wondering why this worked at all before. Anyway, explitly casting this to int should not hurt...